### DISCUSSION TITLE
how to using data from remote server in a component

### DISCUSSION THREAD

**Discussion by jaliy** [GrapesJS/grapesjs#6153] [Q&A] [answered]
I'm writing a list component and each item is load from remote server, so how could I to use data from rest API, many thanks~

**Comment by mohamedsalem401** [comment, marked as answer]
Hey @jaliy, 
It seems you're looking to integrate dynamic data into your GrapesJS components. A new feature will be available soon that allows you to do this.
GrapesJS's DataSources allow you to manage and inject data into your components, styles, and traits programmatically. This means you can dynamically bind data from your REST API to your list component's elements.

It's still work on progress but you can read about it more here https://github.com/GrapesJS/grapesjs/blob/c9aec5cbe45dd0db26a1b2e962a4a0aed2cf9bc1/docs/modules/DataSources.md

**Reply by jaliy** [reply]
Thats great!!! May I know when will this feature will be released ?
